internal/sdk/cloudian/sdk.go

```diff
@@ -346,75 +269,45 @@ func (client Client) DeleteGroupRecursive(ctx context.Context, groupId string) e
 
 // Deletes a group if it is without members.
 func (client Client) DeleteGroup(ctx context.Context, groupId string) error {
-	resp, err := client.doRequest(ctx, http.MethodDelete, "/group",
-		map[string]string{"groupId": groupId}, nil)
-	if err != nil {
-		return err
-	}
-
-	defer resp.Body.Close() // nolint:errcheck
+	_, err := basicCloudianRequest(ctx, client).
+		SetQueryParams(map[string]string{"groupId": groupId}).
+		Delete("/group")
 
-	switch resp.StatusCode {
-	case 200:
-		return nil
-	default:
-		return fmt.Errorf("DELETE unexpected status. Failure: %d", resp.StatusCode)
-	}
+	return err
 }
```

Comment:
Why does `DeleteGroup` not check status code, but `DeleteUserCredentials` do?

Reply:
A slip! fixed..